redis.conf

```diff
@@ -1841,6 +1841,25 @@ client-output-buffer-limit pubsub 32mb 8mb 60
 #
 # client-query-buffer-limit 1gb
 
+# In some scenarios client connections can hog up memory leading to OOM
+# errors or data eviction. To avoid this we can cap the accumulated memory
+# used by all client connections (all pubsub and normal clients). Once we
+# reach that limit connections will be dropped by the server freeing up
+# memory. The server will attempt to drop the connections using the most 
+# memory first. We call this mechanism "client eviction".
+#
+# Client eviction is configured using the maxmemory-clients setting as follows:
+# 0 - client eviction is disabled (default)
+#
+# A positive memory value can be used for the client eviction threshold,
+# for example:
+# maxmemory-clients 1g
+#
+# A negative value (between -1 and -100) means the client eviction threshold
+# is set based on a percentage of the maxmemory setting. For example to set
+# client eviction at 5% of maxmemory:
+# maxmemory-clients -5
```

Comment:
this needs to be updated with the percentage config.
also, maybe mention it in the top comment.

Reply:
thanks, done.